src/main/java/io/confluent/connect/jdbc/source/TimestampIncrementingTableQuerier.java

```diff
@@ -249,6 +258,22 @@ private boolean isIntegralPrimitiveType(Object incrementingColumnValue) {
            || incrementingColumnValue instanceof Byte;
   }
 
+  /**
+   * @param columnName column name
+   * @param columnAlias optional column alias
+   * @param quoteString string used to quote SQL identifiers
+   * @return the quoted column with a prefixed alias, if applicable
+   */
+  private String getQuotedColumn(String columnName, String columnAlias, String quoteString) {
+    String result = "";
+
+    if (!columnAlias.isEmpty()) {
+      result += JdbcUtils.quoteString(columnAlias, quoteString) + ".";
```

Comment:
I think it would be possible, but a bit tricky. We'd have to extend the config to support pre-quoted values in the config. This might not be that bad -- we can check if it is quoted by just checking the first and last chars. Then it's just a matter of being able to unquote and split it up so we can get just the table name so we can process the JDBC ResultSet.